enclave/src/main/java/tech/pegasys/pantheon/enclave/Enclave.java

```diff
@@ -76,8 +76,8 @@ private Request buildPostRequest(
     try (Response response = client.newCall(request).execute()) {
       return objectMapper.readValue(response.body().string(), responseType);
     } catch (IOException e) {
-      LOG.error("Enclave failed to execute ", request);
-      throw new IOException("Enclave failed to execute post", e);
+      LOG.error("Enclave failed to execute {}", request);
+      throw new IOException("Enclave failed to execute post after 3 tries");
```

Comment:
If there is a specific reason to not wrap the actual exception for getting a full traceback, will you add a comment?